lib/openstudio-aws.rb

```diff
@@ -16,13 +16,16 @@
 rescue LoadError
   puts 'Failed to load AWS-SDK-CORE gem'
   puts '  try running: gem install aws-sdk-core'
-  exit
+
+  exit 1
 end
 
+require 'openstudio/core_ext/hash'
 require 'openstudio/lib/openstudio_aws_logger'
 require 'openstudio/aws/aws'
 require 'openstudio/aws/config'
 require 'openstudio/aws/version'
 require 'openstudio/lib/openstudio_aws_instance'
 require 'openstudio/lib/openstudio_aws_wrapper'
+require 'openstudio/lib/openstudio_cloud_watch'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.